compiler/extccomp.nim

```diff
@@ -943,6 +943,7 @@ proc jsonBuildInstructionsFile*(conf: ConfigRef): AbsoluteFile =
   result = getNimcacheDir(conf) / conf.outFile.changeFileExt("json")
 
 proc writeJsonBuildInstructions*(conf: ConfigRef) =
+  # xxx use std/json instead, will result in simpler, more maintainable code.
```

Comment:
I prefer to have and use `std/jsonbuilder` instead. It makes no sense to setup the JsonNode graph just to render it back into a string later.

Reply:
=> https://github.com/nim-lang/Nim/pull/18100#issuecomment-854078280